src/Chainweb/Pact/Backend/ChainwebPactDb.hs

```diff
@@ -341,9 +340,8 @@ doKeys d = do
     pb <- use bsPendingBlock
     mptx <- use bsPendingTx
 
-    let memKeys = DL.toList $
-                  fmap (B8.unpack . _deltaRowKey) $
-                  collect pb `DL.append` maybe DL.empty collect mptx
+    let memKeys = fmap (B8.unpack . _deltaRowKey) $
+                  collect pb ++ maybe [] collect mptx
```

Comment:
You can left-align the `$` here like the code right after, but just a minor consistency suggestion.

```suggestion
    let memKeys = fmap (B8.unpack . _deltaRowKey)
                $ collect pb ++ maybe [] collect mptx
```